### ISSUE TITLE
I installed with yarn install but type update did not work

### ISSUE THREAD

**Issue by jonasmedeiros** [shahen94/react-native-switch#51]
I installed the library with yarn install ... but the type config did not come the index.d.ts and json modification, I did on my own the modification, but for other developers that would break the project that is in Typescript, can you please check

**Comment by Poeticode** [comment]
Yeah, the react-native-switch NPM package is outdated. At some point, could you bump up the version number and publish the latest updates @shahen94 ?
